documentation/API_documentation/QoD_User_Story.md

```diff
@@ -3,8 +3,8 @@
 | **Item** | **Details** |
 | ---- | ------- |
 | ***Summary*** | As an application developer belonging to an enterprise, I want to request (using my application server/backend service), stable latency or stable throughput for a specified App-Flow from a Communication Service Provider (CSP), so that I can ensure better quality of experience for our end users including under network congestion.  |
```

Comment:
Minor suggestion, as it is no longer either stable latency or stable throughput, but not both
```suggestion
| ***Summary*** | As an application developer belonging to an enterprise, I want to request (using my application server/backend service), stable latency and/or stable throughput for a specified App-Flow from a Communication Service Provider (CSP), so that I can ensure better quality of experience for our end users including under network congestion.  |
```